simcon_project/simcon_project/settings.py

```diff
@@ -140,7 +140,7 @@
 STATIC_URL = '/static/'
 MEDIA_URL = '/media/'
 LOGIN_REDIRECT_URL = 'redirect-from-login'
-LOGOUT_REDIRECT_URL = 'login'
+LOGOUT_REDIRECT_URL = '/'
```

Comment:
Hmm, I kind of like logout going to the login screen...

Other opinions?

Reply:
I think that's a good idea too

Reply:
wasnt sure where to direct it but will fix!